File: testdata/typescript/basic/input/service.ts
interface OrderRequest {
  id: string;
  amount: number;
  items: LineItem[];
}

interface ProcessResult {
  success: boolean;
  orderId?: string;
}

export function processOrder(order: OrderRequest, retry: boolean = false): ProcessResult {
  return { success: true };
}

function calculateTotal(items: LineItem[]): number {
  return 0;
}

File: testdata/python/basic/input/service.py
from typing import List, Optional

class OrderRequest:
    id: str
    amount: float
    items: List[str]

class ProcessResult:
    success: bool
    order_id: str

def process_order(order: OrderRequest, retry: bool = False) -> ProcessResult:
    return ProcessResult()

def _calculate_total(items: List[str]) -> float:
    return 0.0
